src/telliot_feeds/sources/manual_sources/diva_manual_source.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from typing import List
+
+from telliot_feeds.datasource import DataSource
+from telliot_feeds.dtypes.datapoint import datetime_now_utc
+from telliot_feeds.dtypes.datapoint import OptionalDataPoint
+from telliot_feeds.utils.log import get_logger
+
+
+logger = get_logger(__name__)
+
+
+def _helper(asset_collateral: str) -> float:
```

Comment:
The use of `collateral` in variable names in this function is confusing, because you use it to get the price of the reference asset & the collateral token